flutter/lib/src/sentry_privacy_options.dart

```diff
@@ -41,6 +41,20 @@ class SentryPrivacyOptions {
       mask: false,
       name: 'SentryUnmask',
     ));
+    // TODO: check for Flutter version and make this testable
```

Comment:
https://github.com/getsentry/sentry-dart/blob/9e18a68bc7594a36254c4731523e8390f38fd4ff/dart/lib/src/event_processor/enricher/flutter_runtime.dart#L44 should be at least `3.33.0-0.2.pre` or higher